lesson_02/src/test.py

```diff
@@ -0,0 +1,48 @@
+import os
+import time
+import requests
+from util import get_base_dir
+
+BASE_DIR = get_base_dir()
+
+if not BASE_DIR:
+    print("BASE_DIR environment variable must be set")
+    exit(1)
+
+JOB1_PORT = 8081
+JOB2_PORT = 8082
+
+RAW_DIR = os.path.join(BASE_DIR, "raw", "sales", "2022-08-09")
+STG_DIR = os.path.join(BASE_DIR, "stg", "sales", "2022-08-09")
+
+
+def run_job1():
```

Comment:
The port and host params not need hardcoded in code, it is bed practice. Could be move to function params and settings.
example:
"""
run_job1(port, host)
""